ptftplib/notify.py

```diff
@@ -58,43 +59,43 @@ class StreamEngine(logging.StreamHandler):
     """Simple stream log handler, similar to what you get using
     logging.basicConfig."""
 
-    def __init__(self, stream, loglevel, format):
+    def __init__(self, stream, loglevel, format_):
```

Comment:
Ah, thanks; I wasn't sure if it was one. I guess I'd rather use `fmt` as the variable name then.